package.json

```diff
@@ -14,15 +14,21 @@
     "compile": "tsc"
   },
   "devDependencies": {
+    "@types/invariant": "^2.2.29",
+    "@types/prop-types": "^15.5.2",
     "@types/react": "^15.0.23",
     "@types/react-redux": "^4.4.40",
     "@types/react-router-redux": "^5.0.0",
+    "invariant": "^2.2.2",
     "typescript": "^2.3.2"
   },
   "peerDependencies": {
-    "react": "^15.5.4",
+    "react": "^15.0.0-0 || ^16.0.0-0",
     "react-redux": "^5.0.4",
-    "react-router-redux": "^5.0.0-alpha.6",
-    "redux": "^3.6.0"
+    "react-router-redux": "^5.0.0-alpha.6"
+  },
+  "dependencies": {
+    "history": "^4.7.2",
+    "prop-types": "^15.6.0"
```

Comment:
I'm not keen on adding prop-types, sorry. I just think type checking is nothing that should happen at runtime.

If I wouldn't harm the people who use TS I would say who cares, but this way everyone using the package bundles prop-types unless she/he configures the bundler to strip them.

Reply:
Well, I agree with you at the point that the typescript checking is better than prop-types, but there are two main points:
1. It is a library that can be used by a non-TS-user, and he/she expects by default that there are `prop-types`, because it is a standard in the React world. It would be pretty confusing if he/she get an error just due to lack of `prop-types`. 
2. You said that if there is `prop-types`, user should configure bundler to strip prop-types. But it is necessary anyway since `react-router-redux-dom-link` is about to be used with `react-router`, and `react-router` uses `prop-types` as well.

Reply:
Okay `react-router` using `prop-types` is a good argument. Let's got with `prop-types`.